PyPi.org push action not working

Username/password auth no longer supported on PyPI/testPyPI (https://github.com/LFPy/LFPy/actions/runs/12865690800/job/35866740261):
```
25hINFO     Response from https://test.pypi.org/legacy/:                           
         403 Forbidden                                                          
INFO     <html>                                                                 
          <head>                                                                
           <title>403 Username/Password authentication is no longer supported.  
         Migrate to API Tokens or Trusted Publishers instead. See               
         https://test.pypi.org/help/#apitoken and                               
         https://test.pypi.org/help/#trusted-publishers</title>                 
          </head>                                                               
          <body>                                                                
           <h1>403 Username/Password authentication is no longer supported.     
         Migrate to API Tokens or Trusted Publishers instead. See               
         https://test.pypi.org/help/#apitoken and                               
         https://test.pypi.org/help/#trusted-publishers</h1>                    
           Access was denied to this resource.<br/><br/>                        
                                                                                
                                                                                
                                                                                
          </body>                                                               
         </html>                                                                
ERROR    HTTPError: 403 Forbidden from https://test.pypi.org/legacy/            
         Forbidden                                                              
Error: Process completed with exit code 1.
```

--- a/.github/workflows/pythonpublish.yml
+++ b/.github/workflows/pythonpublish.yml
@@ -20,16 +20,10 @@ jobs:
         python -m pip install --upgrade pip
         pip install setuptools twine
     - name: Build and publish to Test PYPI
-      env:
-        TWINE_USERNAME: ${{ secrets.TEST_PYPI_USERNAME }}
-        TWINE_PASSWORD: ${{ secrets.TEST_PYPI_PASSWORD }}
       run: |
         python setup.py bdist_wheel
-        twine upload  --repository-url https://test.pypi.org/legacy/ dist/* --verbose
+        twine upload -u __token__ -p ${{ secrets.GH_TESTPYPI_LFPY_TOKEN }} --repository-url https://test.pypi.org/legacy/ dist/* --verbose
     - name: Build and publish
-      env:
-        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
       run: |
         python setup.py bdist_wheel
-        twine upload dist/* --verbose
+        twine upload -u __token__ -p ${{ secrets.GH_PYPI_LFPY_TOKEN }} dist/* --verbose

--- a/.gitignore
+++ b/.gitignore
@@ -133,6 +133,7 @@ dmypy.json
 
 # macos stuff
 .DS_Store
+__MACOSX
 
 # misc
 examples/hoc_combos*

--- a/LFPy/version.py
+++ b/LFPy/version.py
@@ -1,1 +1,1 @@
-version = "2.3.4"
+version = "2.3.5"

--- a/conda_environment.yml
+++ b/conda_environment.yml
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3
+  - python=3.12
   - c-compiler
   - cxx-compiler
   - Cython

